### PR TITLE
Make AdsSymbol even more pythonic

### PR DESCRIPTION
**Pull request by stlehmann**
The purpose of this PR is to make the great AdsSymbol class even more Pythonic by:

* replacing setter/getter functions by properties
* Allowing immediate writing of the value to the plc if auto_update is True

### CODE REVIEW

**Comment by stlehmann** [comment]
@RobertoRoos I made some minor API-changes to make AdsSymbol class more pythonic. You might want to have a look at it and tell me if you approve it.

File: pyads/symbol.py [hunk 1]
```diff
@@ -288,3 +270,44 @@ def get_type_from_str(type_str: str) -> Any:
...
+    @value.setter
+    def value(self, val: Any) -> None:
+        """Set the current value of the symbol.
+
+        If auto_update is True then the the write command will be called automatically.
+
+        """
+        self._value = val
+        # write value to plc if auto_update is enabled
+        if self._auto_update_handle is not None:
```
Comment (RobertoRoos) [inline pyads/symbol.py:312]: I think `if self.auto_update` it's a little easier to read.

**Comment by RobertoRoos** [comment]
I am personally not the biggest fan of magic properties, but I understand this is considered good practice.

Placed a teeny-tiny comment, I like your changes. Thank you for asking me.

File: pyads/symbol.py (hunk 1 above)
Comment (stlehmann) [reply pyads/symbol.py:312]: That's a good suggestion. I implemented the auto_update property after the value property so I didn' t see this. I will change it.

**Comment by stlehmann** [comment]
@RobertoRoos thanks for having a quick look at it :)

**Pull request merged**